src/EditorFeatures/VisualBasicTest/Classification/SemanticClassifierTests.vb

```diff
@@ -543,5 +543,21 @@ end sub
 
             Await TestInClassAsync(text)
         End Function
+
+        <WorkItem(14688, "https://github.com/dotnet/roslyn/issues/14688")>
+        <Fact, Trait(Traits.Feature, Traits.Features.Classification)>
+        Public Async Function OperatorsWithDefaultValuesAreNotAllowed() As Task
+            Dim text =
+<code>
+Public Class Foo
+    Public Shared Operator Not(Optional x As Foo) As Foo
+    End Operator
+End Class
+</code>.NormalizedValue()
+
+            Await TestAsync(text,
```

Comment:
Shouldn't Foo be classified three times? Even if Optionals aren't allowed, I wouldn't expect it to fall down that badly.